shell/platform/darwin/ios/framework/Source/vsync_waiter_ios.h

```diff
@@ -15,6 +15,8 @@
 
 @interface DisplayLinkManager : NSObject
 
+@property(class, nonatomic, readonly) bool minimumFrameDurationDisabled;
```

Comment:
Can you add a comment here?  What does it mean to disable this?  Also might be easier to understand as a positive: `minimumFrameDurationEnabled`

```suggestion
@property(class, nonatomic, readonly) BOOL minimumFrameDurationDisabled;
```